internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,69 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
 	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
 	// TODO: реализовать функцию
+
+	var (
+		errInvalidFormat = errors.New("data format error")
+		errNumberParsing = errors.New("step count retrieval error")
+		errTimeParsing   = errors.New("time value retrieval error")
+		errZeroSteps     = errors.New("step count <= 0")
+		errTimeZero      = errors.New("time value <= 0")
+	)
+	units := strings.Split(datastring, ",") // разделение строки data на слайс строк
+	if len(units) != 2 {                    // проверка длины слайса
+		return fmt.Errorf("%w: expected format - 'number,duration'", errInvalidFormat)
+	}
+	number, err := strconv.Atoi(units[0])
+	if err != nil {
+		return fmt.Errorf("%w: %v", errNumberParsing, err)
```

Comment:
Оборачивать лучше основную ошибку, если хочешь как у тебя
```suggestion
		return fmt.Errorf("%v: %w", errNumberParsing, err)
```
Или как обычно это делают
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```